playwright-e2e/dsm/component/tables/participant-list-table.ts

```diff
@@ -75,6 +76,37 @@ export class ParticipantListTable extends Table {
     return cell.innerText();
   }
 
+  /**
+   * Use to verify that certain headers are currently displayed in the participant list
+   * @param checkDefaultFilterOfStudy choose whether to check for the default filter of a given study - use in combination with studyName param
+   * @param studyName the name of the study whose default filter should be checked
+   * @param customFilter for use to check display of custom filters - this is a given list of column headers that make up the given filter
+   */
+  public async assertDisplayedHeaders(opts: { checkDefaultFilterOfStudy?: boolean, studyName?: StudyName, customFilter?: Label[] }): Promise<void> {
```

Comment:
nit: We ought to start writing code for business logics inside the test script. Writing business logic functions inside a page object file is bad practice.